### PR TITLE
Megeme 17

### PR DESCRIPTION
**Pull request by Balogh08**
Please review my GET "/pitches" endpoint.

### CODE REVIEW

**Review by szirmi — DISMISSED** [review]

File: ASP.NET-Core-Webapp/CustomErrorMessage.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    internal class CustomErrorMessage
+    {
+        private string v;
+
+        public CustomErrorMessage(string v)
+        {
+            this.v = v;
+        }
+    }
+}
```
Comment (fabianzs) [inline ASP.NET-Core-Webapp/CustomErrorMessage.cs]: You shouldn't have modified this.

File: ASP.NET-Core-Webapp/Startup.cs [hunk 2]
```diff
@@ -141,6 +140,11 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, Applicat
 
             }
 
+            if (env.EnvironmentName == "Testing")
+            {
+                //app.UseMiddleware<AuthenticatedTestRequestMiddleware>();
+            }
+
```
Comment (fabianzs) [inline ASP.NET-Core-Webapp/Startup.cs]: Please remove these lines!

File: ASP.NET-Core-Webapp.IntegrationTests/Fixtures/TestContext.cs [hunk 3]
```diff
@@ -12,6 +13,7 @@ public class TestContext : IDisposable
 
         public TestContext()
         {
+            var projectDir = System.IO.Directory.GetCurrentDirectory();
```
Comment (fabianzs) [inline ASP.NET-Core-Webapp.IntegrationTests/Fixtures/TestContext.cs]: Please remove this line!

**Review by fabianzs — COMMENTED** [review]
Please remove TestStartup and remove Configuration folder!

**Review by JohnLackland — APPROVED** [review]
Nice!

**Review by fabianzs — APPROVED** [review]

**Pull request merged**